### ISSUE TITLE
Display only major browser version number when prompting user to confirm the version at the start of a test run session

### ISSUE THREAD

**Issue by ccanash** [w3c/aria-at-app#810]
This issue is related to the version confirmation dialog that appears when a tester activates a link on the test queue page to open a test plan in the test runner.

It seems that the app is saving the major version number but detecting the full version number. The information presented in the browser version field in the dialog is confusing because .0.0 is appended to the major version. That gives users the impression that the full version is expected and saved. But that is not the case.

If the app is only saving the major version number, it should not display .0.0 in the version number field. This is probably one reason for the incorrect alert.

_Originally posted by @mcking65 in https://github.com/w3c/aria-at-app/issues/797#issuecomment-1736555503_

**Comment by mcking65** [comment]
@evmiguel  wrote:

>The app saves the full browser version in the database, which is why .0.0 is being shown in the version number field.

For Chrome, for example, 121.0.0 is not a valid full version number. There is no such version as far as I can tell. So, if the full version is saved, why is a number like 121.0.0 shown as the saved number? The number should be something like 121.0.6167.185.

>This is the current behavior implemented, but were you looking to have the major version saved instead of the full version?

In this issue I proposed saving and showing only the major version number. However, if we are already saving the full version number, then an alternative is to continue to save the full version number and then show the full version number in the app. 

Continuing to show an invalid, non-existent version is confusing.

It remains to be determined how important the browser version is to the ARIA-AT work. So, perhaps the the original intent I expressed in this issue is not the best course of action. In the absence of clear evidence that the full version is not important, it is probably safer to keep storing it if it does not create additional work to do so.
